app/controllers/comments_controller.rb

```diff
@@ -9,6 +12,37 @@ def create
       redirect_to post_path(@post.id)
     end
   end
+  def edit
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+    render "posts/show"
+  end
+  def update
+    @post = Post.find(params[:post_id])
+    @comment = Comment.find(params[:id])
+
+    if @comment.update(comment_params)
+      flash[:notice] = "コメントを更新しました"
+      redirect_to post_path(params[:post_id])
```

Comment:
細かいですが、以下の方がスッキリして見えます

```suggestion
      redirect_to post_path(@post.id)
```